File: 80.lv/integrations/Python/http_client.py
import http.client

conn = http.client.HTTPSConnection("api_dev.apitube.io")

headers = {
    'X-ApiTube-Key': "***KEY***"
}

conn.request("GET", "/v1/sets/46796339?limit=250&offset=0", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: 802.lv/integrations/C#/HttpClient.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
	Method = HttpMethod.Get,
	RequestUri = new Uri("https://api_dev.apitube.io/v1/sets/46796339?limit=250&offset=0"),
	Headers =
	{
		{ "X-ApiTube-Key", "***KEY***" },
	},
};
using (var response = await client.SendAsync(request))
{
	response.EnsureSuccessStatusCode();
	var body = await response.Content.ReadAsStringAsync();
	Console.WriteLine(body);
}
